source/jsonify.d.ts

```diff
@@ -8,9 +8,9 @@ import type {IsAny} from './is-any';
 // Note: The return value has to be `any` and not `unknown` so it can match `void`.
 type NotJsonable = ((...arguments_: any[]) => any) | undefined | symbol;
 
-type JsonifyTuple<T extends [unknown, ...unknown[]]> = {
-	[Key in keyof T]: T[Key] extends NotJsonable ? null : Jsonify<T[Key]>;
-};
+type JsonifyTuple<T extends unknown[]> = T extends [infer F, ... infer R]
+	? [Jsonify<F>, ...JsonifyTuple<R>]
```

Comment:
```suggestion
	? [Jsonify<F>, ...Jsonify<R>]
```

Reply:
@MichaelDeBoey this is actually more general and something I wanted to switch to in our own types in Remix. And it gets the tests passing for me locally.

Reply:
@pcattori we now get `["some value", ...never[]]` for the newly added test, is that expected?